packages/stencil/src/components/datacom-avataravatar/datacom-avataravatar.css

```diff
@@ -0,0 +1,43 @@
+:host {
+  display: block;
+}
+
+.avatar {
+  display: flex;
+  font-family: 'Montserrat';
```

Comment:
You should reference the variable defined in the light.css file for this font-family. That'll ensure that the font family used in the components are all consistent.